lib/datadog.js

```diff
@@ -205,6 +215,11 @@ exports.init = function datadog_init(startup_time, config, events, log) {
    datadogApiKey = config.datadogApiKey;
    datadogApiHost = config.datadogApiHost;
    datadogPrefix = config.datadogPrefix;
+   datadogTags = config.datadogTags;
+
+    if (datadogTags === undefined || datadogTags.constructor !== Array || datadogTags.length < 1) {
+        datadogTags = "";
```

Comment:
I tried sending an empty string to Datadog and it did not tag the metric, but I'm totally fine with making this an empty array. I'll update the code now.